packages/daemon/__tests__/utils/wallet.test.ts

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { EventTxOutput } from '../../src/types';
+import { prepareOutputs } from '../../src/utils';
+
+/**
+ * @jest-environment node
+ */
+
+describe('prepareOutputs', () => {
+  it('should ignore NFT outputs', () => {
+    const nftOutputs: EventTxOutput[] = [{
+      value: 1,
+      token_data: 0,
+      script: 'OmlwZnM6Ly9pcGZzL1FtTlJtNmhRUDN2MlVMclVOZTJQTTY4V1dRb2EyUmVwY1IxejVUVVdWZmd0bzGs',
+      // @ts-ignore: This type is wrong, we should allow null here in the type
+      decoded: null
```

Comment:
This will be refactored in [this PR](https://github.com/HathorNetwork/hathor-wallet-service/pull/182)